setup.py

```diff
@@ -71,16 +71,18 @@
     'future==0.15.2',
     'futures==3.0.5',
     'google-cloud==0.19.0',
+    'google-api-python-client==1.6.0',
```

Comment:
Thanks so much for catching this!

We absolutely should include these dependencies. The versions we've tested, however are still a bit behind the ones you've selected. Would you be willing to change to match the versions we currently use in our [Dockerfile](https://github.com/googledatalab/datalab/blob/master/containers/base/Dockerfile)?

Those versions would be:

    google-api-python-client==1.5.1
    seaborn==0.7.0
    plotly==1.12.5
    psutil==4.3.0